content/java/concepts/math-methods/terms/toDegrees/toDegrees.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
@sotiriszogos please review the content standards [here](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and the term entry template [here](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md) to properly add metadata to this entry. You can also look at the source of any of the existing term entries to get an idea of what we're looking for.